### PR TITLE
:sparkles: NEW: starting point

### PR DESCRIPTION
**Pull request by Yedidyar**

### CODE REVIEW

File: source/getEntityTypescriptPostgres.ts [hunk 1]
```diff
@@ -1,169 +1,522 @@
...
+	writeFileSync,
+} from 'fs';
+import {green, yellow} from 'kleur/colors';
+import {SqlDatabase, remult} from 'remult';
 
-export function generateModels(
-	connectionString: string,
-	tables: string[],
-	dir = '.',
+const deleteFolderRecursive = path => {
```
Comment (Yedidyar) [inline source/getEntityTypescriptPostgres.ts:13]: i think you should add a warning that we will override

File: source/getEntityTypescriptPostgres.ts [hunk 2]
```diff
@@ -1,169 +1,522 @@
...
+	return current_col;
+}
+
+class DbTable {
+	schema: string;
+	db_name: string;
+	graphql_key: string;
+	class_name: string;
+	foreign_key: string;
+
```
Comment (Yedidyar) [inline source/getEntityTypescriptPostgres.ts:109]: does this handle many to many?

File: source/getEntityTypescriptPostgres.ts [hunk 3]
```diff
@@ -1,169 +1,522 @@
...
+		current_col += ' = ' + defaultVal;
+	}
+
+	return current_col;
+}
+
+class DbTable {
+	schema: string;
+	db_name: string;
+	graphql_key: string;
```
Comment (Yedidyar) [inline source/getEntityTypescriptPostgres.ts:106]: i think we should not tie it to graphql it should be entity like remult semantics

File: source/getEntityTypescriptPostgres.ts [hunk 4]
```diff
@@ -1,169 +1,522 @@
...
+			// TODO fix dbName should be able to take a schema
+			props.push(`// dbName: '${table.db_name}'`);
+			props.push(`sqlExpression: 'public.${table.db_name}'`);
+		} else {
+			props.push(`dbName: '${table.db_name}'`);
+		}
+	}
 
-	let first = true;
+	let defaultOrderBy: string = undefined!;
```
Comment (Yedidyar) [inline source/getEntityTypescriptPostgres.ts:264]: why you use null assrtion

**Review by Yedidyar — COMMENTED** [review]
looking good starting point

File: source/getEntityTypescriptPostgres.ts (hunk 1 above)
Comment (jycouet) [reply source/getEntityTypescriptPostgres.ts:13]: Yes, would be good to have a flag to write everything or not. Should we add some tasks? or?

File: source/getEntityTypescriptPostgres.ts (hunk 2 above)
Comment (jycouet) [reply source/getEntityTypescriptPostgres.ts:109]: Nop

File: source/getEntityTypescriptPostgres.ts (hunk 3 above)
Comment (jycouet) [reply source/getEntityTypescriptPostgres.ts:106]: You are right, the naming is not correct. It's called "key" in remult, it's this param: 
image

`key` is probably good enough ✌️

**Pull request merged**